letsencrypt/cli.py

```diff
@@ -1139,6 +1142,27 @@ def main(cli_args=sys.argv[1:]):
 
     return args.func(args, config, plugins)
 
+def check_config_sanity(args):
+    """Validate command line options and display error message if
+    requirements are not met.
+
+    :param args: Command line options
+    :type args: :class:`argparse.Namespace`
+
+    """
+    # Domain checks
+    if args.domains is not None:
```

Comment:
you're right no `--domain` on the CLI results in `args.domains` being `None`; sorry!